### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by Colassal**
Added a warning to the top of the advanced section
Added multiple bullet points to the advanced section
Corrected shade sanity strength to 0.04

### CODE REVIEW

**Comment by Colassal** [comment]
Fixed branch conflicts created by github auto-writing in the file at places other text was supposed to exist. Should be good to merge upon approval.

File: how-to-play/index.md [hunk 1]
```diff
@@ -28,8 +28,6 @@ When the ghost wants to try to kill you, it enters a hunting mode. Your flashlig
 
 After that, it will move through the house and if it is near you, could try and kill you!
 
-**During the hunt the radio on your shoulder does not work at all.**
```
Comment (imagitama) [inline how-to-play/index.md:31]: Why did you remove this?

File: how-to-play/index.md [hunk 2]
```diff
@@ -110,10 +108,27 @@ You can only recover sanity by using Sanity Pills.
...
+  
+  - When looking at a manifested ghost you lose sanity at the rate of `deltatime * ghost_sanity_strength` where all ghosts have a `ghost_sanity_strength` of 0.02 but Shade has 0.04 (It is unknown how or if this is changed at higher difficulties).
+  - Sanity can never drop below 50% in Setup Phase.
+  - Sanity drains slower in Setup Phase (0.09 vs 0.12).
+  - Sanity meters in the van only update every 2 seconds.
+  - Sanity meters in the van fluctuate from -2 to +3 of your actual sanity. EX: If I have 80 sanity the display could show between 78 and 83 sanity.
+  
+  # Equipment Information
+  
+  - Crucifix has a range of 4 meters for all ghosts except the Banshee where the crucifix will have a 6 meter range.
```
Comment (imagitama) [inline how-to-play/index.md:123]: I've heard this is 3m. Can you confirm it is 4m?

**Review by imagitama — COMMENTED** [review]
Thanks for the contribution! Today I made new sub-pages for Sanity and Hunting Mode so I manually applied the changes in your PR.

File: how-to-play/index.md (hunk 1 above)
Comment (Colassal) [reply how-to-play/index.md:31]: Didn't mean to May have gotten overwritten by the auto generated text that caused the pull error

File: how-to-play/index.md (hunk 2 above)
Comment (Colassal) [reply how-to-play/index.md:123]: The code says 4m in it, i can check again later to see if maybe it was changed.

Comment (imagitama) [reply how-to-play/index.md:123]: It is 3m in this patch https://discordapp.com/channels/435431947963990026/661365069006897154/760599252480163840
